### ISSUE TITLE
Auto size modal height

### ISSUE THREAD

**Issue by marcodpt** [chenjiahan/vodal#15]
Is there a way of modal height be computed with the slot html?
My main problem is that for my use i cannot pass a fix height for the modal, because vary a lot depend on the form that user will trigger!

**Comment by zacharytyhacz** [comment]
I agree, I was just having this issue by using a modal to preview a video, though the videos can be different sizes/dimensions.

You can try fiddling around with the class styles as a work-around to try:
```css
.vodal-dialogue {
    /* .... */
}
```
However, I do agree with you that the size properties should have an `auto` value to be given to just fit the contents within.

Something like this maybe?
```html
<vodal :show="true" :height="'auto'" :width="'auto'" ... ... ...></vodal>
```
Another note that I would like to see is a `max_width` / `max_height`, however this can be done in CSS, though not programmatically depending on certain conditions if it were to change.

**EDIT** 
Having something like this implemented may help out [this issue](https://github.com/chenjiahan/vodal/issues/13) as well that is also a problem I have ran into in the past with using these vodals:

**Comment by weblabio** [comment]
+1 
while the height is static - the plugin cannot be used :(

**Comment by zacharytyhacz** [comment]
> +1
> while the height is static - the plugin cannot be used :(

Have you tried overriding with CSS ?

**Comment by fabionixweb** [comment]
.vodal-dialog {
    height: max-content !important;
}
